src/_path.h

```diff
@@ -358,7 +358,7 @@ void get_path_collection_extents(agg::trans_affine &master_transform,
                                  agg::trans_affine &offset_trans,
                                  extent_limits &extent)
 {
-    if (offsets.dim(0) != 0 && offsets.dim(1) != 2) {
+    if (offsets.size() != 0 && offsets.dim(1) != 2) {
```

Comment:
I'm not sure if this needed changing, but it doesn't hurt aside from a vague feeling of asymmetry.